server/pages/index.jade

```diff
@@ -61,6 +61,9 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 					span(class=['environment', 'is-docs'])
 						a(href=devDocsURL title='DevDocs') docs
 
+		if 'development' === env
```

Comment:
i think we want this to be `!== 'development`? errors from dev would just be noise... or is this for testing?